Formula/open-mesh.rb

```diff
@@ -15,6 +15,11 @@ class OpenMesh < Formula
 
   depends_on "cmake" => :build
 
+  patch do
+    url "https://graphics.rwth-aachen.de:9000/OpenMesh/OpenMesh/commit/c5cfef87427a793268f9e012856872bbed958d92.diff"
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission (if this is not an upstream commit, it's not clear) and regardless add an explanation of why the patch is needed in a comment in the formula file. Thanks!